docker-compose.yaml

```diff
@@ -21,29 +20,19 @@ services:
       PAYMENT_QUEUE_ADDRESS: payment

```

Comment:
In David's "without base file" version, he'd moved these environment variables to a separate file. I like that approach as it reduces the duplication of the postgres settings, and makes the file look tidier, at the expense of moving settings into different files (so not all settings in one place).
https://github.com/DEFRA/ffc-demo-claim-service/pull/26/files
Have a look and let me know what you think about the approach.

Reply:
I think if the env vars were shared across files and all env vars are in files, it's a good idea, but  as this is the only place they are used I'd prefer the simple approach of keeping them in line.